lib/security.dart

```diff
@@ -28,21 +29,27 @@ class _TwoFactorEnableScreenState extends State<TwoFactorEnableScreen> {
   final formKey = GlobalKey<FormState>();
   final txtController = new TextEditingController();
 
-  void submit() {
+  void _submit() {
     var cs = formKey.currentState;
     if (cs != null && cs.validate()) {
       Navigator.pop(context, txtController.text);
     }
   }
 
-  void copyKey() {
+  void _copyKey() {
     Clipboard.setData(ClipboardData(text: widget.setup.key));
     snackMsg(context, 'copied key');
   }
 
-  void cantLaunch(String? url) {
-    snackMsg(context, 'no app registered to import two factor data',
-        category: MessageCategory.Warning);
+  void _launchImport() {
+    if (UniversalPlatform.isWeb)
+      html.window.open(
+          'otpauth://totp/Bitforge:${widget.userEmail}?secret=${widget.setup.key}&issuer=Bitforge',
```

Comment:
Should we remove all the hyphens from the set up key?

```dart
"otpauth://totp/Bitforge:${widget.userEmail}?secret=${widget.setup.key.replaceAll('-','')}&issuer=Bitforge",

```

Reply:
test to see if removing the hyphens works with google auth and MS auth and if it does make a PR